src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+    Hello, world!
+  </body>
+</html>
```

Comment:
<img width="214" alt="image" src="https://user-images.githubusercontent.com/55802630/172107699-7134ea97-8a28-4edf-b5b7-0d5f2f53d99d.png">
git doesn't like it 😅